src/ray/gcs/CMakeLists.txt

```diff
@@ -30,7 +30,18 @@ add_custom_command(
   TARGET gen_gcs_fbs
   COMMAND ${FLATBUFFERS_COMPILER} -p -o ${PYTHON_OUTPUT_DIR} ${GCS_FBS_SRC}
   DEPENDS ${FBS_DEPENDS}
-  COMMENT "Running flatc compiler on ${GCS_FBS_SRC}"
+  COMMENT "Running flatc compiler on ${GCS_FBS_SRC} for python"
+  VERBATIM)
+
+# Generate Java bindings for the flatbuffers objects.
+set(RAY_HOME ${CMAKE_CURRENT_LIST_DIR}/../../..)
```

Comment:
If you add `RAY_HOME`, you can also apply it to `set(PYTHON_OUTPUT_DIR ${CMAKE_CURRENT_LIST_DIR}/../../../python/ray/core/generated/)`.